core/src/main/java/org/apache/shiro/session/Session.java

```diff
@@ -149,7 +149,7 @@
      * It is common for a {@code Subject} implementation to retain authentication state in the
      * {@code Session}.  If the session
      * is explicitly stopped by application code by calling this method directly, it could clear out any
-     * authentication state that might exist, thereby effectively &quot;unauthenticating&quot; the {@code Subject}.
+     * authentication state that might exist, thereby effectively &quot;deauthenticating&quot; the {@code Subject}.
```

Comment:
I think the original _might_ be more correct. Though it's possible this line needs to be reworded

Reply:
Fwiw, this one wasn't automated, I actually read through it and thought about it, but I'm absolutely flexible for all changes / totally willing to drop things.

Reply:
Fwiw, this seems ok to me. Should I apply it?

Reply:
Yes, please.